data/en/listevery.json

```diff
@@ -5,15 +5,15 @@
 	"member": "list.listEvery(closure [, delimiter] [, includeEmptyFields] [, multiCharacterDelimiter] [, parallel] [, maxThreads])",
 	"returns": "boolean",
 	"related": [],
-	"description": "",
+	"description": "This function calls a given closure/function with every element in a given string list and returns true, if all of the closure calls returns true.",
 	"params": [
-		{"name": "list", "description": "", "required": true, "default": "", "type": "string", "values": []},
-		{"name": "closure", "description": "", "required": true, "default": "", "type": "function", "values": []},
-		{"name": "delimiter", "description": "", "required": false, "default": ",", "type": "string", "values": []},
-		{"name": "includeEmptyFields", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "multiCharacterDelimiter", "description": "", "required": false, "default": "true", "type": "boolean", "values": []},
-		{"name": "parallel", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "maxThreads", "description": "", "required": false, "default": "20", "type": "numeric", "values": []}
+		{"name": "list", "description": "String list to iterate.", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "closure", "description": "Function/closure that implements the following constructor [function(any value, numeric index, string list, string delimiter):boolean].", "required": true, "default": "", "type": "function", "values": []},
+		{"name": "delimiter", "description": "Delimiter used to separate the string list.", "required": false, "default": ",", "type": "string", "values": []},
+		{"name": "includeEmptyFields", "description": "Specifies whether to include empty fields or not.", "required": false, "default": "false", "type": "boolean", "values": []},
+		{"name": "multiCharacterDelimiter", "description": "Specifies whether the 'delimiter' parameter specifies a multi-character delimiter. If this parameter is true, the 'delimiter' parameter defines a single delimiter consisting of multiple characters.", "required": false, "default": "true", "type": "boolean", "values": []},
```

Comment:
To be consistent I would reverse the statement: "wether the delimiter is a single-character delimiter. Than you don't have to mention that multi-character delimiter is the `false` value here.

Reply:
Wait, **`true`** is the value for the multi-character delimiter.

I'd even make it more precise, like "Specifies whether the 'delimiter' parameter is interpreted as a multi-character delimiter. If this parameter is true, the 'delimiter' parameter defines a single delimiter consisting of multiple characters, otherwise the characters are interpreted as multiple delimiters."

Reply:
That sounds confusing to me but this might happen because I'm not a native speaker. I would say something like this:

"If it is set to `true` all characters are considered as one delimiter; otherwise each character is a delimiter on its own."

Reply:
Agree, that's easier to read. Changed it accordingly in https://github.com/foundeo/cfdocs/pull/871/commits/d6c0c6f588b8b965aa36e76e077772e7c8b2f8cb.